tests/snuba/api/endpoints/test_organization_group_index.py

```diff
@@ -488,6 +488,51 @@ def test_advanced_search_errors(self, mock_record):
                 organization_id=self.organization.id,
             )
 
+    def test_assigned_to_pagination(self):
```

Comment:
I tried running this with your changes commented out and it still passes, could you modify it to capture this case?